static/schemas/2.0/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "workflowGraphs.2.0",
+    "definitions": {
+        "tasks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "oneOf": [
+                    {"required": ["taskName"]},
+                    {"required": ["taskDefinition"]}
+                ],
+                "properties": {
+                    "label": {
+                        "type": "string",
+                        "minLength": 1
+                    },
+                    "taskName": {
+                        "type": "string",
+                        "minLength": 1
+                    },
+                    "taskDefinition": {
+                        "type": "object"
```

Comment:
Since taskDefinition also has strict structure, so here need the sub-schema for the `taskDefinition`. You can either point to an existing external task definition schema (I don't know whether Swagger supports reference external schema) or add the sub-schema here.